src/bindgen/utilities.rs

```diff
@@ -41,6 +41,53 @@ pub fn find_first_some<T>(slice: &[Option<T>]) -> Option<&T> {
 
 pub trait SynItemFnHelpers: SynItemHelpers {
     fn exported_name(&self) -> Option<String>;
+
+    fn swift_name(&self, type_name: Option<&syn::Ident>) -> String;
```

Comment:
Seems like this should belong to our IR instead of syn... Why is this here?

Reply:
You're right, this doesn't need to be here. Moved it into `bindgen::ir::Function`.